src/algebra/star/basic.lean

```diff
@@ -394,6 +395,17 @@ begin
   rw [ring.inverse_non_unit _ ha, ring.inverse_non_unit _ (mt is_unit_star.mp ha), star_zero],
 end
 
+instance invertible.star {R : Type*} [monoid R] [star_monoid R] (r : R) [invertible r] :
```

Comment:
can this safely be an instance?

Reply:
[`invertible`](https://leanprover-community.github.io/mathlib_docs/algebra/invertible.html#invertible) has a random selection of instances, so I think it's probably ok.